apix-impl/config/alfresco-global.properties

```diff
@@ -5,3 +5,6 @@
 #aps.rest.base_url = http://process-service:8080/activiti-app/api
 #aps.rest.username = admin
 #aps.rest.password = admin
+
+#Properties to disable the org.alfresco.repo.node.ContentPropertyRestrictionInterceptor.java since it is botched
+contentPropertyRestrictions.enabled=false
```

Comment:
We've added this just for 6.2. Would it be reasonable to make this propertiesfile exclusive to that version of would that be to much effort. Mostly thinking of unexpected side effects in older versions?

Reply:
I see. I think that from your commits then?
Can you explain a bit more? What was the problem you saw, and where did you find this fix?

Reply:
Looking at it again, a functional change wont be necessary. 
The referenced interceptor was added in 6.2 (hence no impact on older versions). I remembered incorrectly that it was changed.
What it tries to address is a 'gap' in security where a user that is not allowed to change the content of a node, could still change the content url property, and so circumvent the restriction. It does this by straight up disallowing updates to the contenturl property. It was intended to have a whitelist property, but this is bugged into a not working state.
Since apix creates and changes nodes through the contenturl property, it needs this disabled.

Reply:
Tnx Robrecht. I'll add that.